.gitignore

```diff
@@ -12,6 +12,18 @@ compile_commands.json
 CTestTestfile.cmake
 build
 
+###IDE
+# clion
+.idea
+
+# eclipse
+.cproject
+.project
+qtcreator
```

Comment:
This one is a separate IDE, though I am not sure that is the canonical settings path they use? Has been a while since I've last used it.

Reply:
What do you suggest? Removing some lines?

Reply:
If somebody has QtCreator installed and can check quickly, we can look more closely at it, but for now, I would suggest removing it. There are other files that might be helpful (https://github.com/github/gitignore/blob/218a941be92679ce67d0484547e3e142b2f5f6f0/Qt.gitignore), but since nobody of us seems to be using it, and Ginkgo is not a Qt application, I guess we can ignore those.

Reply:
OK